CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
```

Comment:
The `salary` field is redundant, improperly named (violates PascalCase convention), and conflicts with the `Salary` property. Leaving this in the code can cause confusion, serialization issues, or bugs. It should be removed before merging.

```suggestion
// TODO: Remove before merging
// public string salary;  // Employee compensation - stores amount
```